zipline/finance/controls.py

```diff
@@ -425,3 +425,48 @@ def validate(self,
         """
         if _account.leverage > self.max_leverage:
             self.fail()
+
+
+class MinLeverage(AccountControl):
+    """
+    AccountControl representing a limit on the minimum leverage allowed
+    by the algorithm after a threshold period of time.
+    """
+
+    def __init__(self, min_leverage, deadline):
+        """
+        max_leverage is the gross leverage in decimal form. For example,
+        2, limits an algorithm to trading at most double the account value.
+        """
+        super(MinLeverage, self).__init__(min_leverage=min_leverage,
+                                          deadline=deadline)
+        self.min_leverage = min_leverage
+        self.deadline = deadline
+
+        if min_leverage is None:
```

Comment:
You might want to look at using the decorators in `zipline.utils.input_validation` to implement these checks. I think `expect_types(min_leverage=(int, float), deadline=datetime)` and `expect_strictly_bounded(min_leverage=(0, None))` would capture the behavior we want here.  If you do end up using those functions, you probably want to pass `__funcname='MinLeverage'` so that the generated error messages reference `MinLeverage` instead of `__init__`.